src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.md' {
```

Comment:
What is the utility of this ?

Reply:
This was required while reading `.md` files from typescript. Now, since we are using posts.json file this is not required. Removed it